src/node_data.js

```diff
@@ -127,10 +127,15 @@ const getData = function(node) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const importNode = function(node) {
+  let data = node['__incrementalDOMData'];
```

Comment:
Went with that since I didn't need the return value. Is it odd that `importNode` returns our private `NodeData` instances? I think I'd like to remove the return value.